### PR TITLE
[Work in progress] Load the gmfThemes in gmf.searchDirective via the $injector

### PR DESCRIPTION
**Pull request by fredj**
Simplifies the search example

### CODE REVIEW

File: contribs/gmf/examples/search.js [hunk 1]
```diff
@@ -17,25 +17,22 @@ var app = {};
 /** @type {!angular.Module} **/
 app.module = angular.module('app', ['gmf']);
 
-app.module.constant('gmfTreeUrl', 'data/themes.json');
```
Comment (sbrunner) [inline contribs/gmf/examples/search.js]: No mode defined?

**Comment by fgravin** [comment]
Why not

**Comment by fredj** [comment]
Note: a loadTheme() in gmf.SearchController is probably needed

**Comment by fgravin** [comment]
Why are you doing all of this for the search firective ? (init the layermanager, load themes etc..)

**Pull request closed without merging**